include/exadg/postprocessor/error_calculation.cpp

```diff
@@ -119,59 +114,40 @@ ErrorCalculator<dim, Number>::setup(dealii::DoFHandler<dim> const &   dof_handle
   mapping     = &mapping_in;
   error_data  = error_data_in;
 
+  time_control.setup(error_data_in.time_control_data);
+
   if(error_data.analytical_solution_available && error_data.write_errors_to_file)
     create_directories(error_data.directory, mpi_comm);
 }
 
 template<int dim, typename Number>
 void
 ErrorCalculator<dim, Number>::evaluate(VectorType const & solution,
-                                       double const &     time,
-                                       int const &        time_step_number)
+                                       double const       time,
+                                       bool const         unsteady)
 {
+  AssertThrow(
+    error_data.analytical_solution_available,
+    dealii::ExcMessage(
+      "Function can only be called if analytical solution is given. If you don't have an analytical solution you probably accidently set time_control.is_active or didn't check if the function should be called."));
```

Comment:
From a user perspective, I would like to avoid this redundancy. I think a user having to write an application.h file is irritated about these two parameters and their meaning.

Reply:
I will check if the analytical solution field was set instead

Reply:
[52282d1](https://github.com/exadg/exadg/pull/262/commits/52282d18bb5e741a06b37576a457006882c8a534)